aiosqlite/tests/smoke.py

```diff
@@ -313,11 +313,35 @@ def one_arg(num):
                 row = await res.fetchone()
                 self.assertEqual(row[0], 20)
 
-    async def test_create_function_deterministic(self):
+    async def test_create_function_deterministic_pre38(self):
+        """Make sure the deterministic parameter cannot be used in old Python versions"""
+        if sys.version_info >= (3, 8):
```

Comment:
This can be done using [`unittest.skipf`](https://docs.python.org/3/library/unittest.html#unittest.skipIf) e.g.

```python
@unittest.skipif(sys.version_info >= (3, 8), "Tests behaviour on Python versions < 3.8")
async def test_create_function_deterministic_pre38(self):
    # ...
```

Reply:
Scrolling down a bit showed some code using `skipIf` and `skipUnless`, so I did that, thanks!

Reply:
Awesome :)